### PR TITLE
[docs] Document shiny new badge in the docs

### PR DESCRIPTION
**Pull request by ammaraskar**
@inferno-chromium PTAL

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Should probably hold off on this until #2690 is resolved IMO

File: docs/getting-started/new_project_guide.md [hunk 1]
```diff
@@ -400,3 +400,19 @@ Use the [ClusterFuzz web interface](https://oss-fuzz.com/) to review the followi
 * Fuzzer performance analyzer (linked from fuzzer statistics)
 
 **Note:** Your Google Account must be listed in [project.yaml](#projectyaml) for you to have access to the ClusterFuzz web interface.
+
+### Status Badge
+
+![Example
+Badge](https://oss-fuzz-build-logs.storage.googleapis.com/badges/curl.svg)
+
+Once your project has started building, we'd love it if you added our badge in
```
Comment (inferno-chromium) [inline docs/getting-started/new_project_guide.md]: s/building/fuzzing ?

File: docs/getting-started/new_project_guide.md [hunk 2]
```diff
@@ -400,3 +400,19 @@ Use the [ClusterFuzz web interface](https://oss-fuzz.com/) to review the followi
...
+your project's README. This allows you to check your oss-fuzz integration
```
Comment (inferno-chromium) [inline docs/getting-started/new_project_guide.md]: s/oss-fuzz/OSS-Fuzz

File: docs/getting-started/new_project_guide.md (hunk 1 above)
Comment (jonathanmetzman) [reply docs/getting-started/new_project_guide.md]: I think the badge won't be created until the first build though?

Comment (inferno-chromium) [reply docs/getting-started/new_project_guide.md]: ah ok, can we hyperlink "building" to our build page - (https://oss-fuzz-build-logs.storage.googleapis.com/index.html)

**Comment by jonathanmetzman** [comment]
The badge goes red on build failure, which may not be the top bug if other bugs get filed. Not ideal, but the best we can do unless we want to do some redirecting server side.

File: docs/getting-started/new_project_guide.md (hunk 1 above)
Comment (jonathanmetzman) [reply docs/getting-started/new_project_guide.md]: Done

**Comment by inferno-chromium** [comment]
> The badge goes red on build failure, which may not be the top bug if other bugs get filed. Not ideal, but the best we can do unless we want to do some redirecting server side.

Yes right, we can have a handler on oss-fuzz.com, but that is work and can skip for now.

**Comment by jonathanmetzman** [comment]
> > The badge goes red on build failure, which may not be the top bug if other bugs get filed. Not ideal, but the best we can do unless we want to do some redirecting server side.
> 
> Yes right, we can have a handler on oss-fuzz.com, but that is work and can skip for now.

Waiting is sort of a problem because if people adopt before we implement this, they probably won't change.

**Comment by inferno-chromium** [comment]
> > > The badge goes red on build failure, which may not be the top bug if other bugs get filed. Not ideal, but the best we can do unless we want to do some redirecting server side.
> > 
> > 
> > Yes right, we can have a handler on oss-fuzz.com, but that is work and can skip for now.
> 
> Waiting to implement this is sort of a problem because if people adopt before we implement this, they probably won't change.

Yes, but we dont have a choice unless someone wants to spend time on handler.

**Pull request merged**